### PR TITLE
CALS-5656 Disable Facility Search/Profile link from CALS if user don’t have Resource Management or CWS Case Management System privileges

### PR DESCRIPTION
**Pull request by sunilb0575**
#### Which User story does this relate to (link)?
-https://osi-cwds.atlassian.net/browse/CALS-5656

#### Brief feature description
-Disable facility search button, when privileges are not one of 'CWS Case Management' or 'Resource Management'

#### Does this PR block/effect other stories?
-No

#### Do any commands needs to be run in order to run the project with this PR?
- When used this JSON for login, facility search button is disabled
```{"user":"RACFID","staffId":"34","roles":["Supervisor"],"county_code":"19","county_name":"Los Angeles","privileges":["Statewide Read","Sealed"]}```
- Now when used below JSON 
```{"user":"RACFID","staffId":"34","roles":["Supervisor"],"county_code":"19","county_name":"Los Angeles","privileges":["CWS Case Management System"]}```

#### Code coverage percent diff as a result of this PR?
-Added RSpec Feature tests

### CODE REVIEW

File: app/services/content/content_service.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+
```
[the remark this answers is not in the document]
Comment (sunilb0575) [reply app/services/content/content_service.rb]: Done.

File: spec/acceptance/rfa_forms/rfa_01b_application_spec.rb [hunk 2]
```diff
@@ -88,10 +93,8 @@
...
     click_button('Save Progress')
     visit page.driver.current_url
-#byebug
-    expect(page.find('#edit_page > div > div > div > div.left-content.col-xs-3.col-sm-3.col-md-3.col-lg-3 > div > div.nav-menu.col-sm-10 > div > div.nav-menu > div:nth-child(2) > div > div > div > div > nav > ul > div > div:nth-child(1) > li > a').text).to eq("alpha0 beta0")
-    expect(page.find('#edit_page > div > div > div > div.left-content.col-xs-3.col-sm-3.col-md-3.col-lg-3 > div > div.nav-menu.col-sm-10 > div > div.nav-menu > div:nth-child(2) > div > div > div > div > nav > ul > div > div:nth-child(2) > li > a').text).to eq("alpha1 beta1")
-    expect(page.find('#edit_page > div > div > div > div.left-content.col-xs-3.col-sm-3.col-md-3.col-lg-3 > div > div.nav-menu.col-sm-10 > div > div.nav-menu > div:nth-child(2) > div > div > div > div > nav > ul > div > div:nth-child(3) > li > a').text).to eq("alpha2 beta2")
-
+    expect(page.find('#edit_page > div > div > div > div.left-content.col-xs-3.col-sm-3.col-md-3.col-lg-3 > div > div.nav-menu.col-sm-10 > div > div.nav-menu > div:nth-child(2) > div > div > div > div > nav > ul > div > div:nth-child(1) > li > a').text).to eq('alpha0 beta0')
```
Comment (anuroop15) [inline spec/acceptance/rfa_forms/rfa_01b_application_spec.rb]: do we need this path check in test case? can this be brought down to just have id's or tag seelctor.

**Comment by Nasir2712** [comment]
@ratneshraval @sunilb0575 the route for search has to be disabled as well right? or it is a separate story?

**Comment by ratneshraval** [comment]
@Nasir2712 Good point. Although I don't think we should/can disable application level routs based on user privilege. We did that when it was an environment level setting. I don't think it is needed for this.

File: spec/acceptance/rfa_forms/rfa_01b_application_spec.rb (hunk 2 above)
Comment (sunilb0575) [reply spec/acceptance/rfa_forms/rfa_01b_application_spec.rb]: Good catch @anuroop15, but this scenario was not added as a part of this story/pr, just removed byebugs.  I will try to change this in my next PR.

File: spec/acceptance/rfa_forms/rfa_01b_application_spec.rb [hunk 3]
```diff
@@ -1,8 +1,14 @@
+# frozen_string_literal: true
+
 require 'rails_helper'
 require 'vcr'
 require 'faker'
 
 RSpec.feature 'RFA01B', js: true do
+  before(:each) do
+    allow_any_instance_of(WelcomeController).to receive(:user_from_session).and_return(FactoryGirl.build(:user))
```
Comment (Nasir2712) [inline spec/acceptance/rfa_forms/rfa_01b_application_spec.rb]: move this line to rails helper so that you shouldn't have to this in every file.

**Pull request merged**